vars/shared-OpenData.var.yml

```diff
@@ -23,7 +23,7 @@ extensions:
       description: "CKAN Extension for Queensland Government Open Data"
```

Comment:
At some point we should fix this description; it's not just Open Data, but also Publications.